From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 28 Aug 2011 21:45:13 +0100
Subject: [PATCH] alpha, gpio: GENERIC_GPIO default must be n

Since GPIOLIB is optional on alpha, GENERIC_GPIO must not be
selected by default.  If GPIOLIB is enabled, it will select
GENERIC_GPIO.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 arch/alpha/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/alpha/Kconfig b/arch/alpha/Kconfig
index 60cde53..8bb9362 100644
--- a/arch/alpha/Kconfig
+++ b/arch/alpha/Kconfig
@@ -51,7 +51,7 @@ config GENERIC_CMOS_UPDATE
         def_bool y
 
 config GENERIC_GPIO
-	def_bool y
+	bool
 
 config ZONE_DMA
 	bool
-- 
1.7.5.4

